benchmarks/rate_limiter/scenario.py

```diff
@@ -0,0 +1,18 @@
+import bm
+
+
+class RateLimiter(bm.Scenario):
+    rate_limit = bm.var(type=int)
+    time_window = bm.var(type=float)
+
+    def run(self):
+        from ddtrace.internal.compat import time_ns
+        from ddtrace.internal.rate_limiter import RateLimiter
+
+        rate_limiter = RateLimiter(rate_limit=self.rate_limit, time_window=self.time_window)
+
+        def _(loops):
+            for _ in range(loops):
+                rate_limiter.is_allowed(time_ns())
```

Comment:
would a fixed set of time stamps make more sense here?

Reply:
Randomness can be problematic in benchmarks, so I'd say if you're seeing an opportunity to reduce randomness it's worth exploring.